src/db/migrations/20171218203245-AddWidgets.ts

```diff
@@ -32,6 +32,14 @@ export default {
                     description: "Collaborate on documents with members of your room.",
                     optionsJson: '{"defaultUrl":"https://scalar.vector.im/etherpad/p/$roomId_$padName"}',
                 },
+                {
+                    type: "whiteboard",
+                    name: "Whiteboard",
+                    avatarUrl: "/img/avatars/whiteboard.png",
+                    isEnabled: true,
+                    isPublic: true,
+                    description: "A whiteboard app embedded in the room.",
```

Comment:
Wouldn't it be good to have an optionsJson like etherpad has to be able to overwrite the default url on widget selection?

Reply:
But nevertheless this file should not be changed because it defines a database migration. A new migration file should be added on top of the existing migrations that only adds the new fields related to this feature to the table. You can look for example at this widget's migration file https://github.com/nordeck/matrix-dimension/blob/72b1e84e10229c4b0e9b5c01e49f0717dcf2d1de/src/db/migrations/20181021164245-AddSpotifyWidget.ts

maybe there is a script to generate new migrations based on a manually changed database or data model? you should ask Travis how he creates new DB migrations.